scripts/solve_network.py

```diff
@@ -85,21 +85,16 @@
 import pandas as pd
 import pypsa
 from _helpers import configure_logging, create_logger
+from linopy import merge
 from pypsa.descriptors import get_switchable_as_dense as get_as_dense
-from pypsa.linopf import (
-    define_constraints,
-    define_variables,
-    get_var,
-    ilopf,
-    join_exprs,
-    linexpr,
-    network_lopf,
-)
+from pypsa.optimization.abstract import optimize_transmission_expansion_iteratively
```

Comment:
Have you locally tested the features here changed? I feel like not all of them are parsed into the CI [which is not good too...]

Reply:
Yes! I have refactored `add_RES_constraints` and tested it's behaviour which has (obviously) required to add some fixes to account for `linopy` grammar.

There may be some advanced stuff which would be nice to investigate in more depth, but on the level of basic functionality it works.